RobotCode/src/main/java/frc/robot/subsystems/IndexerSubsystem.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.SparkPIDController;
+import com.revrobotics.CANSparkBase.ControlType;
+import com.revrobotics.CANSparkBase.IdleMode;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.core.motors.TeamSparkMAX;
+import frc.robot.constants.Constants;
+import frc.robot.constants.IndexerConstants;
+import frc.robot.constants.Ports;
+
+public class IndexerSubsystem extends SubsystemBase
```

Comment:
Rename to FeederSubsystem. Indexer is a general term, but Feeder is clearer and more specific to the actual mechanism.